.circleci/src/jobs/prepare_delivery.yml

```diff
@@ -0,0 +1,38 @@
+parameters:
+  production_delivery:
+    description: Whether builds are delivered to the production store. If set to true, jira and github releases are created.
+    type: boolean
+docker:
+  - image: circleci/node:12.16.3
+resource_class: small
+steps:
+  - checkout
+  - attach_workspace:
+      at: .
+  - restore_environment_variables
+  - restore_cache:
+      name: Restore Yarn Package Cache
+      keys:
+        - 3-yarn-{{ arch }}-{{ checksum "yarn.lock" }}
+        - 3-yarn-{{ arch }}-
+  - run:
+      name: Install Dependencies
+      command: yarn install --frozen-lockfile
+  - save_cache:
+      name: Save Yarn Package Cache
+      key: 3-yarn-{{ arch }}-{{ checksum "yarn.lock" }}
+      paths:
+        - ~/.cache/yarn
+        - node_modules
+  - when:
+      condition: << parameters.production_delivery >>
+      steps:
+        - run:
+            name: Create jira release # The jira project has to have the same name as the github project. Otherwise this has to be configured.
+            command: tools/create-jira-release create-release ${NEW_VERSION_NAME} --private-key ${JIRA_PRIVATE_KEY} --access-token ${JIRA_OAUTH_TOKEN} --consumer-key ${JIRA_CONSUMER_KEY} --project-name ${CIRCLE_PROJECT_REPONAME}
```

Comment:
Isn't the name of the jira project `NATIVE`? This is `CIRCLE_PROJECT_REPONAME` used here?

Reply:
If integreat-react-native-app is alright then I would propose to hardcode it here. CIRCLE_PROJECT_REPONAME doesn't have to be integreat-react-native-app.

Reply:
`NATIVE` is the key while `integreat-react-native-app` is the name, right?. As this will change anyway when switching to a mono repo, I'll keep it as it is for now.